README.md

```diff
@@ -96,6 +97,17 @@ export default {
 
 Alternatively, you can use `openapi-ts.config.js` and configure the export statement depending on your project setup.
 
+### Clients
+
+We provide a variety of possible clients to use when generating your `openapi-ts` client. The following are available:
```

Comment:
Can you add a note that by default we try to infer so there's no need to set this manually? Only set if the result isn't what you expect or want to force a different client